h/presenters.py

```diff
@@ -33,13 +35,13 @@ def truncate(content, length=60):
     link_text = truncate(link_text)
 
     if href and hostname:
-        link = '<a href="{href}" title="{title}">{link_text}</a><br>({hostname})'
-    elif hostname:
-        link = '<a title="{title}">{link_text}</a><br>({hostname})'
+        link = '<a href="{href}" title="{title}">{link_text}</a><br>{hostname}'
     elif href:
         link = '<a href="{href}" title="{title}">{link_text}</a>'
     else:
-        link = '<a title="{title}">{link_text}</a>'
+        link = '<em>Local file:</em> {title}'
+        if hostname:
+            link += '<br>{hostname}'
```

Comment:
I think this is reasonable behavior, in the absence of actual data from our database telling us otherwise.